src/src/DataStructs/SettingsStruct.h

```diff
@@ -567,6 +570,11 @@ class SettingsStruct_tmpl
   int8_t        console_serial_rxpin = DEFAULT_CONSOLE_PORT_RXPIN;
   int8_t        console_serial_txpin = DEFAULT_CONSOLE_PORT_TXPIN;
   uint8_t       console_serial0_fallback = DEFAULT_CONSOLE_SER0_FALLBACK;
+
+  uint8_t CAN_Tx_pin   = -1;
+  uint8_t CAN_Rx_pin   = -1;
+  long    CAN_baudrate = DEFAULT_CAN_BAUDRATE;
+  int     CAN_node_id  = 0;

```

Comment:
I think this belongs in the CAN Controller, where the CAN Importer can get it's pins from, if needed, as the CAN Controller seems t o be a requirement for the importer to function.